### ISSUE TITLE
introduce jbang config

### ISSUE THREAD

**Issue by maxandersen** [jbangdev/jbang#596]
`jbang edit --open=. --live xyz.java` today requires setting JBANG_EDITOR or EDITOR or similar env to have it open correctly.

It would be nice for things that requires persistence that we could offer doing `jbang config set editor code` to allow setting an actual default value for the user. 

should have similar options as git config that does like we do for jbang-catalog and search in parent dirs

**Comment by quintesse** [comment]
Hey when I last suggested this feature you shot it down ;-)

**Comment by maxandersen** [comment]
Yes - at the time it was not right...or my mind wasn't yet convinced. sorry :)

**Comment by quintesse** [comment]
I might still have the code lying around.... ;-)
